### PR TITLE
Fix UIExplorer search bar look on Android

### PR DESCRIPTION
**Pull request by janicduplessis**
Remove the underline color of the TextInput so the search input looks the same as iOS. Also add a missing type import.

Before
image

After:
image

### CODE REVIEW

**Comment by facebook-github-bot-4** [comment]
By analyzing the blame information on this pull request, we identified @ericvicenti and @sahrens to be potential reviewers.

File: Examples/UIExplorer/js/UIExplorerExampleList.js [hunk 1]
```diff
@@ -215,6 +216,7 @@ const styles = StyleSheet.create({
     borderRadius: 3,
     borderWidth: 1,
     paddingLeft: 8,
+    paddingVertical: 0,
```
Comment (sahrens) [inline Examples/UIExplorer/js/UIExplorerExampleList.js]: Is this necessary?

**Comment by sahrens** [comment]
@facebook-github-bot shipit

File: Examples/UIExplorer/js/UIExplorerExampleList.js (hunk 1 above)
Comment (janicduplessis) [inline Examples/UIExplorer/js/UIExplorerExampleList.js]: Yea otherwise the text is not centered vertically.

Comment (janicduplessis) [inline Examples/UIExplorer/js/UIExplorerExampleList.js]: I think android has some padding by default (probably to offset for the underline)

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-5** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3858957) to review internal test results.

**Pull request closed without merging**